packages/lambda-xyz/src/routes/tile.ts

```diff
@@ -84,7 +88,11 @@ export async function tile(req: LambdaContext, xyzData: TileDataXyz): Promise<La
     const latLon = ProjectionTileMatrixSet.get(xyzData.projection.code).tileCenterToLatLon(xyzData);
     req.set('location', latLon);
 
-    const tileSet = await loadTileSet(req, xyzData.name, xyzData.projection);
+    req.set('tileSet', xyzData.name);
+    req.set('projection', xyzData.projection);
```

Comment:
I believe my previous PR added this info into the log lines here.

Reply:
The DRYing up no longer applies because WMTS and tile rendering now take different paths

Reply:
Moved to `TileOrWmts`